### ISSUE TITLE
Allow Cults to be manually arranged, favored cult identified, all collapsed

### ISSUE THREAD

**Issue by claycle** [sun-dragon-cult/fvtt-system-rqg#696]
### More Than One Cutl Poses Display Problems

Related to issue: #681

The cults on the Rune Magic tab need some way for us to manually arrange them quickly, collapsed the common spell lists, and identified which cult is the "favored" cult that should be identified in the top right of the corner of the character sheet.

I noticed that an Odayla character, when he became a member of an additional cult, had Odayla removed from the upper right corner of the character sheet and replaced with the new cult. I could not determine how, apart from being the first cult on the sheet, this cult could be selected by the user explicitly. Additionally, it does not appear that we can move cults up or down in the list on the rune magic page.

**Comment by wake42** [comment]
The cult that is displayed as the "main cult" at the top of the character sheet is the one with the most Rune Points. I assumed that would be the cult you associate yourself with the most. Does that sound reasonable? In case you have more cults than one with the same amount of rune points it's undefined which will be displayed, probably the one you added first, but I haven't tested that.

About organising the cult tab into horizontal tabs, one per cult in case #681. I think that would solve the problem of being overwhelmed by the long rune magic lists. Currently I agree that the cult page is way too long, but if you only see one cult at a time you should have no problem seeing all your rune magic at once. Do you think you would need additional measures to hide some spells?

What I take from this (and correct me if I'm wrong) is that after #681 is done it would be good to be able to sort the cults manually, and not only by number of rune points (like I envisioned it).
I think I will wait with adding more advanced filtering of rune spells until the tabs are in place and one can get a feel for how it looks in practice.

Any feedback on this is welcome!

**Comment by claycle** [comment]
Yeah, that's what I thought (It was choosing based on rune points). In this particular case, the player has invested 1 more RP in the secondary cult as a momentary thing, but I think he wishes the primary cult to remain his primary cult (it's complicated!)

Most RP is a fine choice *most of the time*, but players will be players and YGMV and stuff. It would be nice to just be able to lock a cult as the "top" cult, RPs be damned :-).

**Comment by wake42** [comment]
Updated the issue text and put this as a "todo" case.

**Comment by The-Axel** [comment]
I think it would be good for the sort order to be 

1. Cult Rank (High Priest top, Lay Member bottom), 
2. Rune Points 
3. 3. Alphabetical.

**Comment by wake42** [comment]
I'm closing this issue, since the ordering done in 64fccf1a855d7777b0b1dc5c169cd37415684770 now is prioritized as mentioned above.
